### ISSUE TITLE
Icons missing

### ISSUE THREAD

**Issue by teknowledgist** [mike-ward/Markdown-Edit#245]
When I install v1.33 (after .NET 4.7 and a reboot) on my clean, test, Win7x64 VM (with nothing else installed), I am getting a window with a bunch of boxes instead of icons.  It looks like this:

[![Markdown-Edit toolbar.png](https://s14.postimg.org/n8dmv6goh/Markdown-_Edit_toolbar.png)](https://postimg.org/image/g55rfkb8t/)

**Comment by mike-ward** [comment]
See #54

**Comment by teknowledgist** [comment]
Ha!  I didn't realize you weren't supporting Win7.  (I guess I haven't been using it in Win7.)  Do you have anything indicating that (other than that closed support issue)?  Putting up a "Windows 7 not supported" notice somewhere visible would probably reduce silly issue postings like mine.

Thanks.

**Comment by mike-ward** [comment]
If you followed the thread a bit further you would have found this link:

http://media.askvg.com/downloads/2013/01/Install-Segoe-UI-Symbol-Font-in-Windows-8.zip

Lots of users use MDE on Win 7. I don't support it because I have one computer and it runs Windows 10.

**Comment by teknowledgist** [comment]
I did see that (and it works).  I'd love to include it in the Chocolatey package 
too, but there would be legal issues.  I am going to reference it in the 
description.

My comment is more that the "normal" user is not going to dig through the closed 
issues to discover that Win7 is unsupported.  A short note/link in a more 
prominent place might save you some aggravation dealing with the question and 
users' aggravation in trying to figure out what is wrong.

The "Installing" documentation needed updated to reflect .NET v4.7 requirements 
anyway, so I added a small section for Windows 7.

Hope you approve,
Erich
